tests/lcs/core/interface/go.livecodescript

```diff
@@ -55,14 +55,19 @@ on TestGoCardUnquotedName
 end TestGoCardUnquotedName
 
 local sStackFileName
-command _TestCreateStack pBinary
+command _TestCreateStack pBinary pVisible
```

Comment:
Can you separate your param names with commas please? (here and in `_TestGoInvisibleVisibleStack`)